packages/core-data/src/resolvers.js

```diff
@@ -116,5 +155,7 @@ export function* hasUploadPermissions() {
 		allowHeader = get( response, [ 'headers', 'Allow' ], '' );
 	}
 
-	yield receiveUploadPermissions( includes( allowHeader, 'POST' ) );
+	const key = compact( [ action, resource, id ] ).join( '/' );
+	const isAllowed = includes( allowHeader, method );
+	yield receiveUserPermissions( key, isAllowed );
```

Comment:
At the very least could we add the ticket URL in the comment so it can be tracked/checked on?

Reply:
> Should it be named receiveUserPermission using singular version to reflect that you can provide only one permission at the time?

👍 02d9502cd

> At the very least could we add the ticket URL in the comment so it can be tracked/checked on?

I think this is referring to this thread? https://github.com/WordPress/gutenberg/pull/12378#discussion_r239331728 If so I added a link to the Trac ticket in 
https://github.com/WordPress/gutenberg/commit/cf2d7012f039500091aae18ce23e5698f77fa2e0.